### PR TITLE
feat: add v2beta3

### PR DESCRIPTION
**Pull request by gxthrj**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### CODE REVIEW

**Comment by tokers** [comment]
Can we split this PR to two parts:

1. Add the new version;
2. Delete the old versions;

**Comment by tao12345666333** [comment]
@gxthrj  any updates?

**Review by tao12345666333 — APPROVED** [review]
Great Job!

LGTM! Thanks!

File: pkg/config/config.go [hunk 1]
```diff
@@ -51,14 +51,12 @@ const (
...
 	// in v1.22.
 	IngressExtensionsV1beta1 = "extensions/v1beta1"
-	// ApisixRouteV1 represents apisixroute.apisix.apache.org/v1
-	ApisixRouteV1 = "apisix.apache.org/v1"
-	// ApisixRouteV2alpha1 represents apisixroute.apisix.apache.org/v2alpha1
-	ApisixRouteV2alpha1 = "apisix.apache.org/v2alpha1"
 	// ApisixRouteV2beta1 represents apisixroute.apisix.apache.org/v2beta1
 	ApisixRouteV2beta1 = "apisix.apache.org/v2beta1"
-	// ApisixRouteV2beta2 represents apisixroute.apisix.apache.org/v2beta2
+	// ApisixRouteV2beta2 represents apisixroute.apisix.apache.org/v2beta3
```
Comment (tao12345666333) [inline pkg/config/config.go:56]:
```suggestion
	// ApisixRouteV2beta2 represents apisixroute.apisix.apache.org/v2beta2
```

Comment (gxthrj) [reply pkg/config/config.go:56]: Good catch~

**Comment by tao12345666333** [comment]
ping @tokers @lingsamuel for review, thanks!

**Review by tokers — APPROVED** [review]

**Pull request merged**